Makefile

```diff
@@ -48,7 +48,12 @@ internal/legacy/archives/cacert.pem:
 	mkdir -p internal/legacy/archives
 	wget https://curl.se/ca/cacert.pem -O internal/legacy/archives/cacert.pem
 
-php: $(PHP_BINARY_PATH)
+.PHONY: archive-hashes
+archive-hashes:
+	cd internal/legacy/archives
+	for f in $(ls -1 | grep -v sha256); do shasum -a 256 "$f"  | awk '{print $1}' > "$f".sha256; done
```

Comment:
We should prefer MD5 here, the hash is used for integrity check and not for tamper protection and the faster MD5 algorithm should be enough.

Reply:
I was aiming for hashes not to need computing at runtime at all (https://github.com/platformsh/cli/pull/94#discussion_r1303408451), the performance difference is negligible, and some people like to run automated scanners looking for usage of bad hash algorithms, but I'm fine with MD5 if you prefer.